src/main/java/stsjorbsmod/relics/CustomJorbsModIntStatsRelic.java

```diff
@@ -0,0 +1,80 @@
+package stsjorbsmod.relics;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+
+import java.text.DecimalFormat;
+import java.util.*;
+
+import static stsjorbsmod.JorbsMod.makeID;
+
+public abstract class CustomJorbsModIntStatsRelic extends CustomJorbsModRelic {
+
+    protected ArrayList<Integer> stats;
+    private static String[] PER_STRINGS = CardCrawlGame.languagePack.getUIString(makeID("RelicStats")).TEXT;
+
+    public CustomJorbsModIntStatsRelic(final String id, final AbstractCard.CardColor relicColor, final RelicTier relicTier, final LandingSound landingSound) {
+        super(id, relicColor, relicTier, landingSound);
+        resetStats();
+    }
+
+    public String getStatsDescription() {
+        String statName = DESCRIPTIONS[DESCRIPTIONS.length - 1];
+        return statName + stats.get(0);
```

Comment:
So as implemented, we can only have a single stat per relic? I suppose that's workable for now. There aren't many relics that make sense to track multiple stats.

Reply:
I added a new interface for any relic that wants stats, and implemented it to add stats for Book of Trials.

Reply:
I like having the interface over having a second relic class. I don't like the duplication of code. This is quite unfortunate.